docs/user-guide/cluster.md

```diff
@@ -187,8 +147,4 @@ You can follow the steps to use SQL to play with distributed insertions and quer
     1 row in set (0.03 sec)
     ```
 
-    You can also use MySQL cli to directly connect to the `Datanode`s, to see how data is distributed. In this way, you can verify that the queries from `Frontend` are actually been executed distributed. For more details about distributed querying, please see [this](https://docs.greptime.com/developer-guide/frontend/distributed-querying) document.
-
-### Other Protocols
-
-`Frontend` also supports distribute read and write other protocols, you can follow our ["Supported Protocols"](https://docs.greptime.com/user-guide/supported-protocols/overview) documents for detailed examples.
+    You can also use MySQL cli to directly connect to the `Datanode`s, to see how data is distributed. In this way, you can verify that the queries from `Frontend` are actually been executed distributed. For more details about distributed querying, please see [this](/developer-guide/frontend/distributed-querying) document.
```

Comment:
Datanode already removed the mysql port, we should use the CLI tool to connect it.https://docs.greptime.com/user-guide/cli-repl 
cc @MichaelScofield